maven-plugin-tools-annotations/src/main/java/org/apache/maven/tools/plugin/extractor/annotations/scanner/DefaultMojoAnnotationsScanner.java

```diff
@@ -271,6 +271,12 @@ protected void analyzeVisitors( MojoClassVisitor mojoClassVisitor )
             {
                 MojoAnnotationContent mojoAnnotationContent = new MojoAnnotationContent();
                 populateAnnotationContent( mojoAnnotationContent, mojoAnnotationVisitor );
+
+                if ( mojoClassVisitor.getAnnotationVisitor( Deprecated.class ) != null )
+                {
+                    mojoAnnotationContent.setDeprecated( "" );
```

Comment:
One should use `StringUtils.EMPTY` or alike to indicate that this is not a typo or bug.

Reply:
ok

Reply:
I already thought so